v4l2loopback.c

```diff
@@ -2128,7 +2128,7 @@ static ssize_t v4l2_loopback_write(struct file *file, const char __user *buf,
 
 /* init functions */
 /* frees buffers, if already allocated */
-static int free_buffers(struct v4l2_loopback_device *dev)
+static void free_buffers(struct v4l2_loopback_device *dev)
```

Comment:
if you change the function signature, you must also change it in the forward declaration: https://github.com/umlaeute/v4l2loopback/blob/9b6a7440226185c02ec6fd3b593f0bb71a02c0bd/v4l2loopback.c#L717